docs/electron.rst

```diff
@@ -223,25 +215,30 @@ a sample wrapper for launching app:
   - type: script
     dest-filename: run.sh
     commands:
-      - npm start --prefix=/app/main
+      - zypak-wrapper.sh /app/main/electron-sample-app "$@"
```

Comment:
Should we suggest zypak or `--no-sandbox`?

Reply:
Zypak is in the baseapp and used by most electron applications AFAIK. VSCode is an exception, I think? and I'm not sure why?

Reply:
VSCode has `--no-sandbox` in their (upstream) wrappers. Recently they've tried to remove it, it broke for some users, and they added it back.

Reply:
(Delayed reply but) IMO if you treat Zypak as "sandbox enabled", then the advice should be "follow upstream defaults", i.e.

- If upstream uses sandboxing, then use Zypak
- If upstream uses --no-sandbox, then leave that

Reply:
Upstreams tend to stick with electron defaults (which is "sandbox enabled" since some version), but still a valid point. Can you please open a follow-up PR?